examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,825 @@
+{
```

Comment:
<p>&gt;with the occurrence of given conditions.</p><p> </p><p>Under given conditions</p><p> </p><p>&gt;will be on 2D grids and we are going to update their values </p><p> </p><p>That's confusing, we are not updated the values of the grid, we are updating the values of object defined on the grid.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>